openlibrary/plugins/openlibrary/lists.py

```diff
@@ -54,7 +54,7 @@ def render(self, doc, lists):
         return render_template("lists/lists.html", doc, lists)
 
 class lists_delete(delegate.page):
-    path = r"(/people/\w+/lists/OL\d+L)/delete"
+    path = r"(/people/[^/]+/lists/OL\d+L)/delete"
```

Comment:
Nice! This should also exclude dots and question marks (otherwise .json might break). Maybe let's do  `[\w-]

Reply:
Will that cover everything?  Lisa mentioned something about usernames containing spaces in the issue.  Can usernames have spaces?

Reply:
Might be good to have some username regex pattern constant somewhere.  Too many different username patterns being used now....

Reply:
😬 Spaces?! That shouldn't be allowed. But idk. We could also expand the negative: `[^/?]` . And actually I think dots migh be ok.

Reply:
I think we should not, I pray there isn't a user with a empty username

Reply:
I think `\w-_` is probably fine

Reply:
```suggestion
    path = r"(/people/[\w-_]+/lists/OL\d+L)/delete"
```

Reply:
@cdrini do we need to check for question marks?  Wouldn't a question mark cause the request to resolve to an entirely different path anyway?